agent/proxycfg/naming_test.go

```diff
@@ -8,6 +8,43 @@ import (
 	"github.com/hashicorp/consul/agent/structs"
 )
 
+// TODO(freddy): Needs enterprise test
```

Comment:
reminder

Reply:
Yep, I have this queued up in an ent commit, will cherry-pick it onto the oss->ent merge fix